_docs/reference/telemetry.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: guide
+title: StorageOS Telemetry
+anchor: reference
+module: reference/telemetry
+---
+
+# StorageOS Telemetry
+
+StorageOS collects telemetry from StorageOS clusters via two different methods
+for two different purposes.
+
+## Sentry.io
+
+StorageOS sends crash dumps and information about the StorageOS cluster to sentry.io.
+This information helps our devleopers monitor and fix crashes. This information 
+includes the cluster version, node counts and license information as well as
+any crash dumps.
+
+Information is sent to Sentry.io once per day or when a process inside the
+StorageOS container crashes. All StorageOS clusters with a routable connection
+to the internet will send crash dumps to Sentry using an HTTPS connection. 
```

Comment:
'over tcp/443' - just to be explicit

Reply:
Also - will we honour env variable https_proxy? Worth finding out if Dom knows.

Reply:
@domodwyer Do you know if we honour that?

Reply:
@actionbuddha We do respect env variables. Please see the latest commit for more information.